src/projectUtils.ts

```diff
@@ -0,0 +1,90 @@
+import { getFirebaseProject } from "./management/projects";
+import { RC } from "./rc";
+
+const _ = require("lodash");
+const clc = require("cli-color");
+const marked = require("marked");
+
+const { FirebaseError } = require("./error");
+
+export function getProjectId({
```

Comment:
Please document exported functions - I am unsure what project or projectId _should_ be coming from